src/main/drivers/accgyro/accgyro_spi_bmi270.c

```diff
@@ -125,65 +125,62 @@ typedef enum {
 
 // BMI270 register reads are 16bits with the first byte a "dummy" value 0
 // that must be ignored. The result is in the second byte.
-static uint8_t bmi270RegisterRead(const busDevice_t *bus, bmi270Register_e registerId)
+static uint8_t bmi270RegisterRead(const extDevice_t *dev, bmi270Register_e registerId)
 {
     uint8_t data[2] = { 0, 0 };
 
-    if (spiBusReadRegisterBuffer(bus, registerId, data, 2)) {
+    if (spiReadRegMskBufRB(dev, registerId, data, 2)) {
```

Comment:
SPI API explanatory comments added in `bus_spi.h`, notably:

```
/*
 * Routine naming convention is:
 * 	spi[Read][Write][Reg][Msk][Buf][RB]
 *
 * 		Read: Perform a read, returning the value read unless 'Buf' is specified
 * 		Write Perform a write
 * 		ReadWrite: Perform both a read and write, returning the value read unless 'Buf' is specified
 * 		Reg: Register number 'reg' is written prior to the read being performed
 * 		Msk: Register number is logically ORed with 0x80 as some devices indicate a read by accessing a register with bit 7 set
 * 		Buf: Pass data of given length by reference
 * 		RB:  Return false immediately if the bus is busy, otherwise complete the access and return true
 */
```